v3/lints/rfc/lint_ext_authority_key_identifier_no_key_identifier.go

```diff
@@ -57,7 +59,23 @@ func (l *authorityKeyIdNoKeyIdField) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *authorityKeyIdNoKeyIdField) Execute(c *x509.Certificate) *lint.LintResult {
-	if c.AuthorityKeyId == nil && !util.IsSelfSigned(c) { //will be nil by default if not found in x509.parseCert
+	if util.IsCACert(c) && util.IsSelfSigned(c) {
+		if c.AuthorityKeyId == nil {
+			// There is one exception; where a CA distributes its public key in the form of a "self-signed"
+			// certificate, the authority key identifier MAY be omitted.
+			return &lint.LintResult{Status: lint.Pass}
+		} else if !bytes.Equal(c.AuthorityKeyId, c.SubjectKeyId) {
```

Comment:
Maybe we could also remove this else if at least at this lint? It checks whether the SKI/AKI values are consistent, which is not the focus of this lint. We could consider creating a new lint for this case.

Reply:
I'm amenable to that.